ush/ufsda/stage.py

```diff
@@ -32,9 +32,20 @@ def gdas_fix(input_fix_dir, working_dir, config):
     else:
         case_anl = config['CASE_ANL']
     layers = int(config['LEVS'])-1
-    # link static B files
-    ufsda.disk_utils.symlink(os.path.join(input_fix_dir, 'bump', case_anl),
-                             config['fv3jedi_staticb_dir'])
+
+    # figure out staticb source
+    if config['STATICB_TYPE'] == 'bump':
+        staticb_source = 'bump'
+    elif config['STATICB_TYPE'] == 'identity':
+        staticb_source = 'identity'
+    else:
+        staticb_source = 'gsibec'
+
```

Comment:
couldn't that just be:
```python
staticb_source = config['STATICB_TYPE']
```
perhaps add an error trap too at that level, making sure that 
```python
staticb_source in ['bump','identity',.....]
```
?

Reply:
well to build on this, it would be
`staticb_source = config.get('STATICB_TYPE', 'gsibec')` to include the default value

Reply:
Suggested change to `/ush/ufsda/statge.py` accepted, tested, and committed at 866dd52.    Thank you, @guillaumevernieres and @CoryMartin-NOAA .  I'm still spinning up on best python practices.  I start with verbose and clunky and refine from there.   I appreciate your guidance to compress multiple lines into one.